### DISCUSSION TITLE
Why was Kong Gateway Performance Testing framework removed in version 3.5

### DISCUSSION THREAD

**Discussion by juvodu** [Kong/kong#12193] [Help]
The Docu says performance testing framework was removed in Kong Gateway version 3.5:

> Note: As of Kong Gateway 3.5.x, this feature has been deprecated. It is not supported past 3.4.x.

https://docs.konghq.com/gateway/3.4.x/reference/performance-testing-framework/

Is that correct? And why is it not mentioned in the changelog of Kong Gateway version 3.5?
https://docs.konghq.com/gateway/latest/breaking-changes/

Is there a new recommendation how to easily test performance of Kong Gateway or customer expected to build their own performance testing tooling from scratch?

**Comment by chronolaw** [comment]
@fffonion , could you give us some explain?

**Reply by fffonion** [reply]
Unforutnately I don't have answer to that.
